Header header

float64 Vin
float64 Vout
float64 Iout
float64 Pout
float64 tempurature

float64 dutyCycle
float64 switchingFreq

bool busy
bool off
bool vout_ov_fault
bool iout_oc_fault
bool vin_uv_fault
bool temp_fault
bool cml_fault
bool vout_fault
bool iout_fault
bool pout_fault
bool input_fault
bool pwr_gd
bool fan_fault
bool other
bool unknown
